.github/workflows/compare_storage_layout.py

```
@@ -1,68 +1,81 @@
-    # Combine relevant columns into a single string for comparison
-    df['Combined'] = df[['Name', 'Type', 'Slot', 'Offset', 'Bytes']].apply(lambda row: '|'.join(row.values), axis=1)
-
-    return df['Combined'].tolist()
-
-def compare_layouts(clientChainGateway_entries, bootstrap_entries):
+import subprocess
+
+def parse_output(contract_name, lines):
+    # Clean up the output and create a dataframe
+    data = []
+    separator_line = len(lines);
```

Ruff

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)